docs/doc/21-load-data/index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: OVERVIEW
+slug: ./
+---
+
+Databend enables you to load data from the following locations:
+
+- Stage:  Databend loads data from staged files. A Databend stage can be internal or external. For more information, see [What are Databend Stages](../30-reference/30-sql/00-ddl/40-stage/index.md#what-are-databend-stages).
+
+- Bucket or container: Databend loads data from files stored in an object storage bucket or container. For the object storage solutions supported by Databend, see [Supported Object Storage Solutions](../10-deploy/00-understanding-deployment-modes.md#supported-object-storage-solutions).
+
+- Local file system: Databend loads data from local files.
+
+- Remote server: Databend loads data from remote files (including IPFS) accessible by URLs starting with *HTTPS://*.
```

Comment:
Load from `https` is no different from load from `fs`/`bucket`. They all act as a special `stage`. Can we unify them together?

Reply:
Thanks for help with review. The table lists the data locations Databend supports loading data from. We have a COPY INTO page to indicate that Databend loads from all the locations you mentioned above using COPY INTO.

This is still a draft. I will send you for review when it's done.